### PR TITLE
message stats tests

### PR DESCRIPTION
**Pull request by Sild**

### CODE REVIEW

File: src/test/java/com/axibase/tsd/api/method/message/MessageMethod.java [hunk 1]
```diff
@@ -48,19 +72,47 @@ public static Boolean insertMessage(final Message message) throws Exception {
         return insertMessage(message, 0);
     }
 
+    public static boolean messageExist(final Message message) throws Exception {
+        Map<String, Object> query = new HashMap<>();
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageMethod.java]: Add model object?

File: src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        assertEquals("Message count mismatch", new BigDecimal(DATES.size()), samples.get(0).getV());
+    }
+
+    /**
+     * #2945
+     */
+    @Test
+    public void testAggregateDetail() throws Exception {
+        Map<String, Object> statsQuery = prepareSimpleMessageStatsQuery(MESSAGE_STATS_ENTITY);
+        Map<String, Object> aggregate = new HashMap<>();
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java]: Use model object?

File: src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+        assertEquals("Message count mismatch", new BigDecimal(DATES.size()), samples.get(0).getV());
+    }
+
+    /**
+     * #2945
+     */
+    @Test
+    public void testAggregateUnknownRaiseError() throws Exception {
+        Map<String, Object> statsQuery = prepareSimpleMessageStatsQuery(MESSAGE_STATS_ENTITY);
+        Map<String, Object> aggregate = new HashMap<>();
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java]: Use model object?

File: src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+            insertMessageCheck(message);
+        }
+    }
+
+    /**
+     * #2945
+     */
+    @Test
+    public void testNoAggregate() throws Exception {
+        Map<String, Object> statsQuery = prepareSimpleMessageStatsQuery(MESSAGE_STATS_ENTITY);
```
Comment (GregoryKutuzov) [inline src/test/java/com/axibase/tsd/api/method/message/MessageQueryStatisticsTest.java]: Use model object?

**Comment by GregoryKutuzov** [comment]
Completed review

**Comment by Sild** [comment]
testAggregateUnknownRaiseError should fail follow the #3329 issue

**Comment by rmakulov** [comment]
lgtm

**Pull request merged**